test/fixtures/merchant_fixture.csv

```diff
@@ -13,7 +13,7 @@ id,name,created_at,updated_at
 12, byMarieinLondon, 2016-01-11 11:59:37 UTC, 2015-07-28 04:52:03 UTC
 13, JUSTEmonsters, 2016-01-11 12:17:26 UTC, 1980-11-23 09:11:30 UTC
 14, Uniford, 1999-02-13 02:27:21 UTC, 2011-02-08 11:30:31 UTC
-15, thepurplepenshop, 2016-01-11 18:30:35 UTC, 2084-08-03 01:12:21 UTC
+15, Uniford, 2016-01-11 18:30:35 UTC, 2084-08-03 01:12:21 UTC
```

Comment:
wondering why we have two of these merchants? I think the more variance the better.

Reply:
I wanted to test if we got all the merchants by name.